Applications/FileIO/Legacy/OGSIOVer4.cpp

```diff
@@ -487,31 +475,42 @@ bool readGLIFileV4(const std::string& fname,
     else
         INFO("GeoLib::readGLIFile(): tag #POLYLINE not found.");
 
+    if (!ply_vec->empty())
+        geo.addPolylineVec(std::move(ply_vec), unique_name,
+                           std::move(ply_names));
+
+    // copy of ply_names
+    std::map<std::string, std::size_t> ply_names_copy;
```

Comment:
minor: Better would be a reason for a copy. Otherwise remove the comment---it's already in the variable name.